node/blockchain/utils/lock.py

```diff
@@ -12,7 +12,7 @@
 
 
 def get_pymongo_client():
-    if (client := thread_storage.pymongo_client) is None:
+    if (client := getattr(thread_storage, 'pymongo_client', None)) is None:
```

Comment:
why?

Reply:
I got a case when class "thread_storage" doesn't have "pymongo_client" attribute.

Reply:
you should remove `thread_storage.pymongo_client = None` above then

Reply:
I removed it. Thank you.